src/mailroom.py

```diff
@@ -0,0 +1,122 @@
+"""Mailroom Implementation.  Create thank you cards or donation reports."""
+from __future__ import unicode_literals
+
+donors = {'jordan schatzman': [200, 240, 45, 2, 1000],
+          'rick valenzuela': [500, 300, 100, 50, 1.24],
+          'sally johnson': [50, 100, 1, 2, 99],
+          'jane': [39, 23, 532, 2432],
+          'bob': [89, 23, 12, 3.45643]}
+
+
+def main():
+    """Allow the user to begin the program."""
+    while True:
+        input_prompt = 'If you want to submit a thank you, enter'
+        input_prompt += ' "Send a Thank You".  If you would like to see a'
+        input_prompt += ' donation report, enter "Create a Report".'
+        input_prompt += ' If at any time you wish to return to this menu,'
+        input_prompt += ' enter "menu".  Otherwise, enter "Quit" to quit: '
+        choice = input(input_prompt)
+        if choice.lower() == 'create a report':
+            print (create_a_report(donors))
+        elif choice.lower() == 'send a thank you':
+            control_thank_you()
+        elif choice.lower() == 'quit':
+            exit()
+        else:
+            print ('\n Please submit a valid input. \n')
+            main()
+
+
+def control_thank_you():
+    """Handle logic for sending a thank you."""
+    while True:
+        input_prompt = 'If you want to see a list of donors, enter "list".'
+        input_prompt += '  Otherwise, enter a donor name: '
+        choice = input(input_prompt)
+        if choice == 'list':
+            sorted_donors = create_sorted_list(donors)
+            for donor in sorted_donors:
+                print (donor)
+        elif choice == 'menu':
+            print ('\n \n')
+            main()
+        elif choice.lower() in donors:
+            chosen_donor = choice.lower()
+            input_prompt = choice + ' has already donated.'
```

Comment:
By moving this stuff into global constants, you will clean up the actual functional code and make it more readable.